pkg/kubelet/apis/config/validation/validation_linux.go

```diff
@@ -38,5 +41,19 @@ func validateKubeletOSConfiguration(kc *kubeletconfig.KubeletConfiguration) erro
 		return fmt.Errorf("invalid configuration: singleProcessOOMKill must not be explicitly set to false when using cgroup v1")
 	}
 
+	if userNs := kc.UserNamespaces; userNs != nil {
+		if idsPerPod := userNs.IDsPerPod; idsPerPod != nil {
+			if *idsPerPod < userNsUnitLength {
+				return fmt.Errorf("invalid configuration: userNamespaces.idsPerPod must not be less than %d", userNsUnitLength)
+			}
+			if *idsPerPod%userNsUnitLength != 0 {
+				return fmt.Errorf("invalid configuration: userNamespaces.idsPerPod must be a multiple of %d", userNsUnitLength)
+			}
+			if *idsPerPod >= 1<<32 {
```

Comment:
would this be more readable comparing to math. MaxUint32 in the condition and outputting MaxUint32 in the message instead of 1<<32?

Reply:
done